templates/project/.github/workflows/test.yaml.twig

```diff
@@ -68,13 +68,10 @@ jobs:
               with:
                   {% verbatim %}php-version: ${{ matrix.php-version }}{% endverbatim %}
                   coverage: pcov
-{% if branch.hasService('mongodb') %}
-                  tools: composer:v{{ project.composerVersion }}, pecl
-{# Change mongodb extension version to a stable one when available #}
-                  extensions: mongodb-1.9.0
-{% else %}
-                  tools: composer:v{{ project.composerVersion }}
-{% endif %}
+
+                  tools: composer:v{{ project.composerVersion }}{% if branch.hasService('mongodb') %}, pecl{% endif %}
```

Comment:
What about https://github.com/sonata-project/dev-kit/blob/master/templates/project/.github/workflows/qa.yaml.twig#L34 ?

And we could stop using service mongodb and use tool: pecl instead.

```suggestion
                  tools: {{ ['composer:v' ~ {{ project.composerVersion }}]|merge(branch.tools)|join(, ) }}
```

Reply:
Then you need to remove the `service` option and add a `tools` option (and update the yaml) @wbloszyk

Reply:
> Then you need to remove the `service` option and add a `tools` option (and update the yaml) @wbloszyk

IMO this change should be done in other PR.